### ISSUE TITLE
Add Email or SMS 

### ISSUE THREAD

**Issue by jransom2** [souliss/soulissapp#3]
For any alarm event add Email or SMS the event to another device for self monitoring.  Allows a device to be on the same network using Souliss App and send email or sms text for any alarm event.

**Comment by marksev1** [comment]
This can be added on the Openhab side??

**Comment by plinioseniore** [comment]
You should look for a binding that does this, never looked at.

**Comment by mcbittech** [comment]
Yes, you can do from OH.
Mail working good, sms it's possible,but OH pay for you, and i think you
can use alternative way,avoiding cost to OH.
Recently i've added on my OH configuration Telegram binding.
Work like a charm, faster than SMS, and without money. :))
I think this way is better also than mail, because on Telegram chat you
have your event history.

Hope that helps!

Flavio.
Il 01/apr/2016 14:17, "marksev1" notifications@github.com ha scritto:

**Comment by marksev1** [comment]
Well you need to have data transfer on which could not be good if you are in a foreign country.

**Comment by mcbittech** [comment]
I dont understand very well.
You need an Internet connection in case of mail or telegram.
Explain me better please....

Flavio.
Il 02/apr/2016 09:25, "marksev1" notifications@github.com ha scritto:

**Comment by marksev1** [comment]
Yes exactly, for sms you don't need internet connection. So could be cheaper to receive an sms from home automation controller vs going online (in a foreign country) to check the email.
